tests/wpt/css-tests/selectors-3_dev/xhtml1/css3-selectors-lang-014.xht

```diff
@@ -27,7 +27,7 @@ This tests a detail related to :lang support. If :lang is not supported, a messa
 <script>
 test(function() {
 assert_equals(document.getElementById('colonlangcontroltest').offsetWidth, 0)
-assert_equals(document.getElementById('box').offsetWidth, 50);
+assert_equals(document.getElementById('box').offsetWidth, 100);
```

Comment:
only for `css3-selectors-lang-014` right

both `css3-selectors-lang-009 / 005` are fine